File: Exercicios_Em_Python_CV/desafio_010.py
#Crie um programa que leia o quanto de dinheiro uma pessoa tem na carteira
#e mostre quantos dolares
valor_na_carteira = float(input('Digite quanto você tem na carteira:'))
dolares = valor_na_carteira/5.5
print('Com R${:.2f} você pode compra U${:.2f}'.format(valor_na_carteira,dolares))

File: Exercicios_Em_Java_CV/Desafio010.java
package fundamentos;

import java.util.Scanner;

/*
 * Crie um programa que leia quanto dinheiro uma pessoa tem na
 * carteira e mostre quantos dolares ela pode comprar
 */
public class Desafio010 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.println("Quanto você possui em sua carteira:R$");
		float dinheiro = scanner.nextFloat();
		System.out.printf("O valor R$%s permite comprar U$%.2f", dinheiro, dinheiro/3.27);
		
		scanner.close();
	}

}
